app/controllers/admin/routes.py

```diff
@@ -157,6 +159,76 @@ def rsvpLogDisplay(eventId):
     else:
         abort(403)
 
+@admin_bp.route('/event/<eventId>/renew', methods=['POST'])
+def renewEvent(eventId):
+    try: 
+        formData=request.form
+        try:
+            assert formData['timeStart'] < formData['timeEnd']
+        except AssertionError:
+            flash("End time must be after start time", 'warning')
```

Comment:
When the start time is after the event end time or end time before the start time, it might be better to make the error message `danger` (I believe) instead of `warning` to make it more clear right away that the event was not saved since the modal gets closed

Reply:
The reason this was made as a warning alert instead of a danger alert is that on the create event page, when the start time is after the end time, they also use a warning alert.